### PR TITLE
Update sdk version

### PR DESCRIPTION
**Pull request by IloveJavaa**
Asana - https://app.asana.com/0/0/1205309820635916/f

### CODE REVIEW

File: Assets/Trackier/Android/TrackierAndroid.cs [hunk 1]
```diff
@@ -114,6 +114,119 @@ public static void setUserId(string userId)
...
+}
+
+public static string getP5()
+{
+	AndroidJavaObject androidClass = new AndroidJavaObject("com.trackier.sdk.TrackierSDK");
+	string data = androidClass.CallStatic<string>("getP2");
+	return data;
+}
+
+public static string getClickId()
```
Comment (Hemant-Mann) [inline Assets/Trackier/Android/TrackierAndroid.cs:201]: indentation

File: Assets/Trackier/Unity/TrackierUnity.cs [hunk 2]
```diff
@@ -4,132 +4,324 @@
...
+		if (IsEditor())
+		{
+			return "";
 		}
 
-		public static void setUserEmail(string userEmail)
+		return TrackierAndroid.getAd();
+	}
+
+#endif
```
Comment (Hemant-Mann) [inline Assets/Trackier/Unity/TrackierUnity.cs:120]: remove multiple if endif, make a single if endif

File: exampleApp/Assets/Trackier/Android/TrackierAndroid.cs [hunk 3]
```diff
@@ -6,77 +6,253 @@
...
+{
+public static void Start(string appToken, string environment) 
+{
+	try {
+		AndroidJavaClass unityPlayer = new AndroidJavaClass("com.unity3d.player.UnityPlayer");
+		AndroidJavaObject trackierSDK = new AndroidJavaObject("com.trackier.sdk.TrackierSDK");
+		AndroidJavaObject activity = unityPlayer.GetStatic<AndroidJavaObject>("currentActivity");
+		AndroidJavaObject context = activity.Call<AndroidJavaObject>("getApplicationContext");
+		AndroidJavaObject trackierSDKConfig = new AndroidJavaObject("com.trackier.sdk.TrackierSDKConfig",context,appToken,environment);
+		trackierSDKConfig.Call("setSDKVersion", "1.6.46");
```
Comment (Hemant-Mann) [inline exampleApp/Assets/Trackier/Android/TrackierAndroid.cs:19]: remove

File: exampleApp/Assets/Trackier/Android/TrackierAndroid.cs [hunk 4]
```diff
@@ -6,77 +6,253 @@
...
+}
+		
+public static void setUserName(string userName)
+{
+	try {
+		AndroidJavaClass unityPlayer = new AndroidJavaClass("com.unity3d.player.UnityPlayer");
+		AndroidJavaObject trackierSDK = new AndroidJavaObject("com.trackier.sdk.TrackierSDK");
+		AndroidJavaObject activity = unityPlayer.GetStatic<AndroidJavaObject>("currentActivity");	   
+		trackierSDK.CallStatic("setUserName",userName);  
+	} catch(System.Exception e) {
```
Comment (Hemant-Mann) [inline exampleApp/Assets/Trackier/Android/TrackierAndroid.cs:73]:
```suggestion
	} catch (System.Exception e) {
```

File: exampleApp/Assets/Trackier/Android/TrackierAndroid.cs [hunk 5]
```diff
@@ -6,77 +6,253 @@
...
+		Debug.Log("System.Exception: "+e.Message);
 	}
-#endif
 }
 
+public static void setUserPhone(string userPhone)
```
Comment (Hemant-Mann) [inline exampleApp/Assets/Trackier/Android/TrackierAndroid.cs:78]: indentation

File: exampleApp/Assets/Trackier/Android/TrackierAndroid.cs [hunk 6]
```diff
@@ -6,77 +6,253 @@
...
+{
+	try {
+		AndroidJavaClass unityPlayer = new AndroidJavaClass("com.unity3d.player.UnityPlayer");
```
Comment (Hemant-Mann) [inline exampleApp/Assets/Trackier/Android/TrackierAndroid.cs:81]: unused variable

File: exampleApp/Assets/Trackier/Android/TrackierAndroid.cs [hunk 7]
```diff
@@ -6,77 +6,253 @@
...
+		AndroidJavaObject trackierSDK = new AndroidJavaObject("com.trackier.sdk.TrackierSDK");
+		AndroidJavaObject activity = unityPlayer.GetStatic<AndroidJavaObject>("currentActivity");
```
Comment (Hemant-Mann) [inline exampleApp/Assets/Trackier/Android/TrackierAndroid.cs:83]: unused variable

File: exampleApp/Assets/Trackier/Android/TrackierAndroid.cs [hunk 8]
```diff
@@ -6,77 +6,253 @@
...
+	}
+	catch(System.Exception e) {
+		Debug.Log("System.Exception: "+e.Message);
+	}
+}
+
+public static void setUserId(string userId)
+{
+	try {
+		AndroidJavaClass unityPlayer = new AndroidJavaClass("com.unity3d.player.UnityPlayer");
```
Comment (Hemant-Mann) [inline exampleApp/Assets/Trackier/Android/TrackierAndroid.cs:107]: same as mentioned above

File: exampleApp/Assets/Trackier/Android/TrackierAndroid.cs [hunk 9]
```diff
@@ -6,77 +6,253 @@
...
+}
+
+public static string getAdID()
+{
+        AndroidJavaObject androidClass = new AndroidJavaObject("com.trackier.sdk.TrackierSDK");
+        string data = androidClass.CallStatic<string>("getAdID");
+        return data;
+}
 
+public static string getAdSet()
```
Comment (Hemant-Mann) [inline exampleApp/Assets/Trackier/Android/TrackierAndroid.cs:131]: ??

File: exampleApp/Assets/Trackier/Android/TrackierAndroid.cs [hunk 10]
```diff
@@ -6,77 +6,253 @@
...
+}
+
+public static string getChannel()
+{
+    AndroidJavaObject androidClass = new AndroidJavaObject("com.trackier.sdk.TrackierSDK");
+    string data = androidClass.CallStatic<string>("getChannel");
+    return data;
+}
+
+public static string getP1()
```
Comment (Hemant-Mann) [inline exampleApp/Assets/Trackier/Android/TrackierAndroid.cs:166]: just 1 example is enough

File: exampleApp/Assets/Trackier/Unity/TrackierUnity.cs [hunk 11]
```diff
@@ -39,50 +53,242 @@ public static void setUserName(string userName)
...
+public static string getAdSet()
+        {
+            if (IsEditor())
+            {
+                return "";
+            }
+            return TrackierAndroid.getAdSet();
+        }
+
+#endif
```
Comment (Hemant-Mann) [inline exampleApp/Assets/Trackier/Unity/TrackierUnity.cs:146]: multiple if endif

File: exampleApp/Assets/Trackier/Unity/TrackierUnity.cs [hunk 12]
```diff
@@ -39,50 +53,242 @@ public static void setUserName(string userName)
...
+            if (IsEditor())
+            {
+                return "";
+            }
+            return TrackierAndroid.getCampaignID();
+        }
+#endif
+
+#if UNITY_ANDROID
+public static string getChannel()
```
Comment (Hemant-Mann) [inline exampleApp/Assets/Trackier/Unity/TrackierUnity.cs:186]: why do we need this here? this is the example folder

**Review by Hemant-Mann — CHANGES_REQUESTED** [review]

File: Assets/Trackier/Android/TrackierAndroid.cs (hunk 1 above)
Comment (IloveJavaa) [reply Assets/Trackier/Android/TrackierAndroid.cs:201]: done

File: exampleApp/Assets/Trackier/Android/TrackierAndroid.cs (hunk 4 above)
Comment (IloveJavaa) [reply exampleApp/Assets/Trackier/Android/TrackierAndroid.cs:73]: done

File: exampleApp/Assets/Trackier/Android/TrackierAndroid.cs (hunk 6 above)
Comment (IloveJavaa) [reply exampleApp/Assets/Trackier/Android/TrackierAndroid.cs]: done

File: exampleApp/Assets/Trackier/Android/TrackierAndroid.cs (hunk 8 above)
Comment (IloveJavaa) [reply exampleApp/Assets/Trackier/Android/TrackierAndroid.cs]: done

File: exampleApp/Assets/Trackier/Unity/TrackierUnity.cs (hunk 11 above)
Comment (IloveJavaa) [reply exampleApp/Assets/Trackier/Unity/TrackierUnity.cs:146]: done

File: Assets/Trackier/Unity/TrackierUnity.cs (hunk 2 above)
Comment (IloveJavaa) [reply Assets/Trackier/Unity/TrackierUnity.cs]: done

File: exampleApp/Assets/Trackier/Android/TrackierAndroid.cs (hunk 7 above)
Comment (IloveJavaa) [reply exampleApp/Assets/Trackier/Android/TrackierAndroid.cs]: done

**Pull request merged**